pkgs/development/libraries/libraw1394/default.nix

```diff
@@ -11,6 +11,7 @@ stdenv.mkDerivation rec {
   meta = { 
     description = "Library providing direct access to the IEEE 1394 bus through the Linux 1394 subsystem's raw1394 user space interface";
     homepage = "https://ieee1394.wiki.kernel.org/index.php/Libraries#libraw1394";
-    license = ["GPL" "LGPL"];
+    license = stdenv.lib.licenses.lgpl2;
```

Comment:
For copyright stuff I typically consult Debian. [In this case](http://metadata.ftp-master.debian.org/changelogs//main/libr/libraw1394/libraw1394_2.1.0-3_copyright) I see just some build tools/scripts are using GPL (listed under ``The following files are licensed under GPL Licence``). It's perhaps questionable whether these need to be considered as parts of the source or not...